packages/@aws-cdk-testing/framework-integ/test/aws-ec2/test/integ.vpc-flow-logs.ts

```diff
@@ -118,13 +118,26 @@ class TestStack extends Stack {
   }
 }
 
+class TransitGatewayFlowLogStack extends Stack {
+  constructor(scope: App, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const transitGateway = new CfnTransitGateway(this, 'TransitGateway', {});
+    new FlowLog(this, 'FlowLogsCW', {
+      resourceType: FlowLogResourceType.fromTransitGatewayId(transitGateway.ref),
+      flowLogName: 'TransitGatewayFlowLogName',
+    });
+  }
+}
+
 const featureFlagTest = new FeatureFlagStack(app, 'FlowLogsFeatureFlag');
 
 const integ = new IntegTest(app, 'FlowLogs', {
   testCases: [
     new TestStack(app, 'FlowLogsTestStack'),
     featureFlagTest,
     new DependencyTestStack(app, 'DependencyTestStack'),
+    new TransitGatewayFlowLogStack(app, 'TransitGatewayFlowLogStack'),
   ],
   diffAssets: true,
 });
```

Comment:
Are there assertions you could add here to confirm the FlowLog is created as you intend it to be?

Reply:
I tried it. But, when I ran integration tests, the following error has occurred. The error message `Response object is too` is ambiguous, so I cannot proceed.

```
FlowLogsDefaultTestDeployAssertXXXXXX | 3/5 | 4:31:08 PM |
CREATE_FAILED        | Custom::DeployAssert@SdkCallEC2describeFlowLogs
| FlowLogs/DefaultTest/DeployAssert/AwsApiCallEC2describeFlowLogsXXXXXXXXXXXXX/Default/Default
(AwsApiCallEC2describeFlowLogsXXXXXXXXXXX) Response object is too
long.
```

Could you help me?

Reply:
Yes, this is a common problem and not well documented. One solution is to do something along the lines of:
```
const describe = integ.assertions.awsApiCall(...);
describe.assertAtPath('path.key.in.object', ExpectedResult.stringLikeRegexp('string you expect'));
```
This will prevent the full response object coming back to cloudformation.